hq/markdown/parameter-store.md

```diff
@@ -0,0 +1,57 @@
+# SSM Parameters
```

Comment:
Probably worth mentioning that this uses configraun, which (I think) only applies to EC2 instances (not Lambdas).

Reply:
It doesn't _have_ to use Configraun.